eslint.config.mjs

```diff
@@ -502,8 +504,6 @@ export default typescript.config([
             // Internal packages.
             ['^(sentry-locale|sentry-images)(/.*|$)'],
 
-            ['^(getsentry-images)(/.*|$)'],
```

Comment:
Why this got removed?

Reply:
I merged getsentry images into sentry-images.